dspace-server-webapp/src/main/java/org/dspace/app/rest/GroupRestController.java

```diff
@@ -76,6 +80,9 @@
     @Autowired
     GroupUtil groupUtil;
 
+    @Autowired
+    GroupRestRepository repository;
```

Comment:
You should be able to remove all the changes to `GroupRestController` now that you've refactored this into `GroupParentObjectLinkRepository`